Package.swift

```diff
@@ -6,19 +6,28 @@ import PackageDescription
 let package = Package(
     name: "Apexy",
     platforms: [
-      .macOS(.v10_12),
-      .iOS(.v10),
-      .tvOS(.v10),
+      .macOS(.v10_13),
+      .iOS(.v11),
+      .tvOS(.v11),
       .watchOS(.v4)
     ],
     products: [
-        .library(name: "Apexy", targets: ["Apexy"]),
+        .library(name: "Apexy", targets: ["Apexy_URLSession"]),
+        .library(name: "Apexy_Alamofire", targets: ["Apexy_Alamofire"]),
+        .library(name: "Apexy_RxSwift", targets: ["Apexy_RxSwift"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.2.0"))
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.2.0")),
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "5.0.0")
     ],
     targets: [
-        .target(name: "Apexy", dependencies: ["Alamofire"]),
+        .target(name: "Apexy_RxSwift", dependencies: ["Apexy", "RxSwift"]),
+        .target(name: "Apexy_Alamofire", dependencies: ["Apexy", "Alamofire"]),
+        .target(name: "Apexy_URLSession", dependencies: ["Apexy"]),
```

Comment:
why aren't the words merged ?

Reply:
Initially I used symbol `+` but have some issues with it. And I replaced `+` with `_`. I think it is a good idea to merge the words.